### PR TITLE
[SMALLFIX] Module name rewording.

### PR DESCRIPTION
**Pull request by jsimsa**
This PR changes the minicluster project name from "Mock Tachyon Cluster" to "Tachyon Mock Cluster". This is purely a cosmetic change for consistency with other modules, all of which have names starting with Tachyon.

### CODE REVIEW

**Comment by apc999** [comment]
+1 LGTM

File: minicluster/pom.xml [hunk 1]
```diff
@@ -10,8 +10,8 @@
 
   <artifactId>tachyon-minicluster</artifactId>
   <packaging>jar</packaging>
-  <name>Mock Tachyon Cluster</name>
-  <description>Mock Tachyon Cluster for testing</description>
+  <name>Tachyon Mock Cluster</name>
+  <description>Tachyon Mock Cluster for testing</description>
```
Comment (calvinjia) [inline minicluster/pom.xml]: Since we are doing some renaming, I find it weird that the artifact ID is not the same as the name. (`Mock` vs `Mini`)

Comment (jsimsa) [inline minicluster/pom.xml]: Done.

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**